opendebates/forms.py

```diff
@@ -83,3 +86,41 @@ def save(self, commit=True):
 class OpenDebatesAuthenticationForm(AuthenticationForm):
     username = forms.CharField(max_length=254,
                                label="Username or Email")
+
+
+class MergeFlagForm(forms.ModelForm):
+    duplicate_of_url = forms.URLField()
+
+    class Meta:
+        model = Flag
+        fields = ('duplicate_of_url', )
+
+    def __init__(self, *args, **kwargs):
+        self.idea = kwargs.pop('idea')
+        self.voter = kwargs.pop('voter')
+        super(MergeFlagForm, self).__init__(*args, **kwargs)
+
+    def clean_duplicate_of_url(self):
+        # parse the URL and use Django's resolver to find the urlconf entry
+        path = urlparse(self.cleaned_data['duplicate_of_url']).path
+        url_match = resolve(path)
+
+        if url_match.url_name != 'vote':
+            raise forms.ValidationError('That is not the URL of a question.')
+
+        duplicate_of_pk = url_match.kwargs.get('id')
+        self.duplicate_of = Submission.objects.exclude(pk=self.idea.pk) \
```

Comment:
It looks like we create submissions "approved" (https://github.com/caktus/django-opendebates/blob/65a58ff97ee4f4fe140a8c389d9cd8a56fc32ceb/opendebates/views.py#L310) and never change that in the code, so it must just be a way for admins to pseudo-delete things by settings approved=False. I think we can just treat a submission with approved=False as not existing.